README.md

```diff
@@ -13,22 +13,61 @@ in hence we build a tool using react+nodejs on top of the swarm cluster that wil
 ![Image description](https://github.com/vldocker/dfp-ui/blob/master/first-page.png)
 -------------------------------------------------------------------------------------------------------------------------
  # requirements:
- 
-go to address [PROXY_IP]:[PROXY_PORT]/metrics and defined the environment variables STATS_USER and STATS_PASS
+
+[Docker Flow Proxy](http://proxy.dockerflow.com/) up and running with environment variables `STATS_USER` and `STATS_PASS`
  
 -------------------------------------------------------------------------------------------------------------------------
  # A Docker Service is created with the following syntax: 
- 
- 
+
+The command that follows assumes that [Docker Flow Proxy](http://proxy.dockerflow.com/) is attached to the network `proxy` and that the name of the service is `proxy`. Please change `--network` and `--env` values if proxy is using a different network or the service is created with a different name.
+
+```bash
 service create --name dfp-ui /
---network proxy /
---publish 3333:3333 /
---env PROXY_HOST_AND_PORT="http://proxy:8080"/ 
---constraint 'node.role == manager' /
---mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock /
-dockervoyagerlabs/dfproxy:1.7.4
+    --network proxy /
+    --publish 3333:3333 /
+    --env PROXY_HOST_AND_PORT="http://proxy:8080"/
+    --constraint 'node.role == manager' /
+    --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock /
+    dockervoyagerlabs/dfproxy:1.7.4
```

Comment:
Remove `1.7.4` unless you're sure that you'll remember to update the docs with each new version.